src/machine/behavior/is/memory/ldbu.rs

```diff
@@ -1,6 +1,14 @@
 use machine::state::State;
+use machine::state::mem::ByteAt;
 
-pub fn ldbu(_state: &mut State, _x: u8, _y: u8, _z: u8) {
-    unimplemented!();
-}
+pub fn ldbu(state: &mut State, x: u8, y: u8, z: u8) {
+
+    let op1: u64 = state.gpr[y].into();
+    let op2: u64 = state.gpr[z].into();
+    let mut a = op1 + op2;
+    a = a % 2u64.pow(64);
```

Comment:
A `+` that overflows is considered a bug, even if it doesn't panic. Arithmetic is only checked for overflows in debug builds since it's very costly. The `+` should probably be replaced by `.wrapping_add` or whatever other behaviour is specified (and likewise for all other operations).